Makefile

```
@@ -268,3 +272,25 @@ README.md: docs/ ## Generates README.md based on the contents of the docs direct
 
 download_acuant_sdk: ## Downloads the most recent Acuant SDK release from Github
 	@scripts/download_acuant_sdk.sh
+
+clobber_db:  ## resets the database for make setup
+	bin/rake db:create
+	bin/rake db:environment:set
+	bin/rake db:reset
+	bin/rake db:environment:set
+	bin/rake dev:prime
+
+clobber_assets:  ## removes assets
+	bin/rake assets:clobber
+	RAILS_ENV=test bin/rake assets:clobber
+
+clobber_logs:  ## purges logs
+	rm -f log/*
+	rm -rf tmp/cache/*
+	rm -rf tmp/encrypted_doc_storage
+	rm -rf tmp/letter_opener
+	rm -rf tmp/mails
+
+## Remove assets and logs, and unused gems, but leave DB alone
+tidy: clobber_assets clobber_logs
```

putting the comment at the end lets `make help` read it correctly
```suggestion
tidy: clobber_assets clobber_logs ## Remove assets and logs, and unused gems, but leave DB alone
```

---

On the "It would look better to say nothing" front... Until I saw this comment, I was unaware of `make help`! I thought we just had a pattern of commenting everything. 😇 

Thanks for these comments; I'm going to clean these up and also make the comments read slightly better now that I have this context.